CONTRIBUTING.md

```diff
@@ -569,8 +569,7 @@ Refer datalad/config.py for information on how to add these environment variable
 - *DATALAD_LOG_VMEM*:
   Reports memory utilization (resident/virtual) at every log line, needs `psutil` module
 - *DATALAD_EXC_STR_TBLIMIT*: 
-  This flag is used by the datalad extract_tb function which extracts and formats stack-traces.
-  It caps the number of lines to DATALAD_EXC_STR_TBLIMIT of pre-processed entries from traceback.
+  This flag is used by datalad to cap the number of traceback steps included in exception logging and result reporting to DATALAD_EXC_STR_TBLIMIT of pre-processed entries from traceback.
```

Comment:
minor: not that we are 100% consistent in this file, but generally we do wrap lines in this file.

Reply:
Looking at the file, I really don't know what you're talking about. What do you want me to do?

Reply:
I was thinking about simple split into two lines:
```suggestion
  This flag is used by datalad to cap the number of traceback steps included in exception logging and
  result reporting to DATALAD_EXC_STR_TBLIMIT of pre-processed entries from traceback.
```